viewer/packages/tools/src/Geomap/MapConfig.ts

```diff
@@ -0,0 +1,193 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+/**
+ * Supported map Providers Bing, Here & Mapbox
+ */
+export enum MapProviders {
+  BingMap = 'BingMap',
+  HereMap = 'HereMap',
+  MapboxMap = 'MapboxMap'
+}
+
+/**
+ * Map data for Mapbox
+ */
+export enum MapboxMode {
+  /**
+   * Access the map data using a map style.
+   */
+  Style = 100,
+  /**
+   * Access the map data using a map id.
+   */
+  Map_Id = 101
```

Comment:
Can you elaborate what this means? Maybe link Mapbox doc if thats meaningful

Reply:
Added details & link to mapbox documentation